scripts/results/plot_fanning.py

```diff
@@ -63,3 +63,23 @@
                         topomap_args=dict(vmin=-300, vmax=300))
 fig.set_size_inches(12, 6, forward=True)
 fig.savefig('FanningB.pdf', bbox_to_inches='tight')
+
+###############################################################################
+# Finally, we can also use the tSSS data which is equivalent to doing a
+# highpass filter and the "fanning" will not be visible also in this case.
+from mne.preprocessing import maxwell_filter # noqa
+
+raw_fname = op.join(study_path, 'ds117', subject, 'MEG', 'run_01_raw.fif')
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+
+raw = maxwell_filter(raw, calibration=cal, cross_talk=ctc, st_duration=1.0,
```

Comment:
1 sec is very short. 10 sec is better if it still works

Reply:
hmm ... I went with what the documentation said:

> Spatiotemporal SSS acts as implicitly as a high-pass filter where the cut-off frequency is 1/st_dur Hz

According to this, if you want a cutoff at 1 Hz you need to use a 1.0 s duration. 

I'm not sure what the rationale behind the above logic in the documentation is though. If you use longer windows, it will allow longer drifts? How do these windows overlap?

Reply:
Also note: `st_duration` of 10. doesn't really reduce the fanning and neither did an `st_duration` of 5.

Reply:
There can be an interaction between window duration and st_correlation, perhaps you can lower `st_correlation` when doing `st_duration=10.` to eliminate fanning. Can you see if it's possible? If it is, how do the number of components removed compare between the `st_duration=10` and `st_duration=1` solutions?

Reply:
Is the number of components removed indicated by the message "Projecting 11 intersecting tSSS components for  489.000 -  489.999 sec (#490/494)" ? If so, it seems that `st_duration` doesn't have much effect for higher `st_correlation` but seems to matter more as you lower `st_correlation`. 

I didn't do this too systematically but here is what I observed:

| st_correlation | st_duration | #components | fanning present? | evoked amplitude? |
|--------------------|----------------|--------------------|------------------------|---------------------------|
|       0.98          |       10        |  around 11      |           yes            |               ok              |
|                        |        1         |  around 11      |           yes            |               ok              |
|       0.95          |        10       |  around 11      |           yes            |               ok              |
|                        |         1        | 12-13              |           no              |              ok              |
|       0.8            |        10       |  18-19             |           no              |              ok              |
|                        |         1        |   21-22            |           no              |              not ok        |
|       0.7            |         10      |   21                 |           no              |              ok              |
|                        |         1        |    30                |           no              |              not ok        |

Reply:
Interesting, that's not what I expected. I guess let's stick with 0.95/1.0 then. Thanks for checking

Reply:
that is a very useful table @jasmainak :) I never thought much about tSSS, but we do have the fanning problem a lot on our datasets as well. Seems as we maybe need to reproduce this table for our own datasets. If we ever get around to it, I'll be sure to share with you.

Reply:
Definitely! Thanks @wmvanvliet :)